CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.6] - 2021-03-15
+### Changed
+- Updated Java version for SonarQube Scanner in Travis builds.
```

Comment:
I guess we don't need to make a version up for a such change. You can add it as a fixed issue to version 0.9.5